pages/api/mindmaps/[key]/index.js

```diff
@@ -82,8 +88,15 @@ const MindMapAPI = async (req, res) => {
 
           return res.status(200).json(result)
         }
-
-        return res.status(404).json({ message: 'Not Found.' })
+      case 'POST':
```

Comment:
A PATCH request would be better to signify this is an update, and also align better with the conventions used elsewhere in this codebase.

Reply:
Also, this should go into `pages/api/mindmaps/index.js` since:
1. The key is not read from the path,
2. It becomes consistent with the convention used in the _nodes_ API.